src/Forms/TreeMultiselectField.php

```diff
@@ -115,52 +116,73 @@ public function getSchemaStateDefaults()
 
     /**
      * Return this field's linked items
-     * @return ArrayList|DataList $items
+     * @return SS_List $items
```

Comment:
This is pretty close to an API change - is there any reason to change this to SS_List?  Will it things still work fine as ArrayList|Datalist?

Reply:
The return type hasn't changed, I just simplified the return typehint (common base). I can make it ArrayList|DataList if you like.

Reply:
I have done this.